packages/console/src/components/CodeEditor/index.tsx

```diff
@@ -77,9 +77,9 @@ function CodeEditor({
   };
 
   // TODO @sijie temp solution for required error (the errorMessage is an empty string)
-  const finalErrorMessage = typeof error === 'string' && !!error ? error : t('general.required');
+  const finalErrorMessage = error?.length ? error : t('general.required');
```

Comment:
nit: Just use `error ?? t('general.required')`, or even directly write it in tsx in line 136 without declaring the variable